src/plugin/any23/howto_upgrade_any23.txt

```diff
@@ -0,0 +1,8 @@
+1. Upgrade Any23 dependency in trunk/ivy/ivy.xml
```

Comment:
You can go ahead and upgrade this when you make the update to the any23 2.1 dependency

Reply:
c380e2d